src/integrationTest/java/org/openstreetmap/atlas/geography/atlas/AtlasIntegrationTest.java

```diff
@@ -37,21 +39,32 @@ public static Atlas loadCuba()
 
     protected Atlas loadBahamas(final Polygon polygon)
     {
-        final String path = OsmPbfLoaderIntegrationTest.class.getResource("BHS_07082015.pbf")
```

Comment:
This is the same PBF file, except the old had pre-way-sectioned roads, which threw everything off.